lib/usd/ui/CMakeLists.txt

```diff
@@ -44,7 +44,10 @@ target_sources(${PROJECT_NAME}
 # -----------------------------------------------------------------------------
 target_compile_definitions(${PROJECT_NAME}
     PRIVATE
+        # These flags are needed for Maya
         $<$<BOOL:${IS_MACOSX}>:OSMac_>
+        $<$<BOOL:${IS_LINUX}>:LINUX>
+
```

Comment:
I have noticed a lot of targets are adding these two flags privately.  Since these flags are needed for Maya, it makes more sense I make them as part of PUBLIC section in mayaUsd shared library.  I will have a separate PR to address this but for now it should be ok.